src/lib/sections/Navigation/Banner/Banner.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import { Meta } from "@storybook/react";
+
+import { Navigation } from "@/lib/sections/Navigation/Navigation";
+
+const meta: Meta<typeof Navigation.Banner> = {
+  title: "Sections/Navigation/Banner",
+  component: Navigation.Banner,
+  render: (args) => (
+    <Navigation isCollapsed={false}>
+      <Navigation.Drawer>
+        <Navigation.Header>
+          <Navigation.Banner>
+            {args["children"]}
+          </Navigation.Banner>
+        </Navigation.Header>
+      </Navigation.Drawer>
+    </Navigation>
+  ),
+  tags: ["autodocs"],
+  parameters: {
+    status: {
+      type: "candidate",
+    },
+  },
+};
+
+export default meta;
+export const Example = { args: { children: (
+  <Navigation.Logo>
+    <Navigation.LogoTag>
+      <Navigation.LogoIcon
+        fill="#fff"
+        viewBox="0 0 165.5 174.3"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <ellipse cx="15.57" cy="111.46" rx="13.44" ry="13.3" />
+        <path d="M156.94 101.45H31.88a18.91 18.91 0 0 1 .27 19.55c-.09.16-.2.31-.29.46h125.08a6 6 0 0 0 6.06-5.96v-8.06a6 6 0 0 0-6-6Z" />
```

Comment:
`svg` should be passed as a child

Reply:
I fixed this on the previous PR, however I forgot to push my changes :facepalm: I've cherry picked the commit addressing that. Will push soon